implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -21,21 +23,11 @@ use crate::Result;
 ///  - connects to the Orchestrator with the retrieved token to create a project
 pub async fn enroll_user(app: &AppHandle<Wry>) -> Result<()> {
     let app_state: State<AppState> = app.state::<AppState>();
-    if app_state.state().identities.default().is_err() {
-        info!("creating a default identity");
-        create_default_identity(app_state.state())
-            .await
-            .map_err(|e| {
-                error!("{:?}", e);
-                e
-            })?;
-    };
```

Comment:
This part is not needed, as the vault/identity are initialized when creating the AppState instance in the application setup. Let me know if I'm missing something here @etorreborre

Reply:
I think I was the one missing something :-). Please remove it.